shell/pages/c/_cluster/_product/_resource/create.vue

```diff
@@ -4,9 +4,18 @@ import ResourceDetail from '@shell/components/ResourceDetail';
 export default {
   name:       'ClusterResourceCreate',
   components: { ResourceDetail },
+  data() {
+    return {
+      /**
+       * Inherited global identifier prefix for tests
+       * Define a term based on the parent component to avoid conflicts on multiple components
+       */
+      componentTestid: 'cluster-manager-create'
```

Comment:
See https://github.com/rancher/dashboard/pull/6284/files/fd795f1e9e162f90af27cfbcaedd8944cfd14d06#r945511620. As this is the root page it won't have any component properties passed to it

Reply:
K, I'll switch to data.